render-release-notes.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+me=$(basename $0)
+
+function showUsage() {
+	cat<<-EOF
```

Comment:
the `-` is only needed if you indent the contents, but then you cannot use different indentations... So let's just delete the `-`...